pkgs/applications/networking/cluster/nerdctl/default.nix

```diff
@@ -4,6 +4,14 @@
 , makeWrapper
 , installShellFiles
 , buildkit
+, libselinux
```

Comment:
?

Reply:
I'm guessing because of this

https://github.com/containerd/nerdctl/blob/main/extras/rootless/containerd-rootless.sh#L102-L109

But we can probably leave that to be picked up at runtime